tests/core/test_broker.py

```diff
@@ -47,6 +50,28 @@ def test_pika_message_property_app_id(monkeypatch: MonkeyPatch):
     assert pika_producer._get_message_properties().app_id == rasa_environment
 
 
+@pytest.mark.parametrize(
+    "queue_arg,queues_arg,expected",
+    [
+        ("rasa_core_events", None, ["rasa_core_events"]),  # default case
+        (["q1", "q2"], None, ["q1", "q2"]),  # supplying a list for `queue` works too
+        ("q1", "q2", ["q2"]),  # `queues` arg supplied, takes precedence
+        ("q1", ["q2", "q3"], ["q2", "q3"]),  # same, but with a list
+        (None, "q1", ["q1"]),  # only supplying `queues` works
```

Comment:
Maybe we want to assert something also about whether a `UserWarning` should be shown for each case

Reply:
can we also do a `(None, None,  ["rasa_core_events"])` case where the user didn't provide any configuration?

Reply:
not really because that would override the default argument `queues=("rasa_core_events",)`. I'll add `(None, ["rasa_core_events"], ["rasa_core_events"])` though which is the default case